libraries/AP_HAL_ChibiOS/hwdef/PixPilot-V6PRO/hwdef.dat

```diff
@@ -0,0 +1,285 @@
+# hw definition file for processing by chibios_hwdef.py
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1160
+
+FLASH_SIZE_KB 2048
+
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+
+
+# order of I2C buses
+I2C_ORDER  I2C2 I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART2 USART3 UART4 UART8 UART7 OTG2
+
+#DEFAULTGPIO OUTPUT LOW PULLDOWN
+
+# USB.
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+PA9 VBUS INPUT OPENDRAIN
+PC0 VBUS_nVALID INPUT PULLUP
+
+
+# telem1
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+PD3 USART2_CTS USART2
+PD4 USART2_RTS USART2
+
+PD3 EXTERN_GPIO4 OUTPUT GPIO(4) ALT(1)
```

Comment:
We are not 100% certain this works....test thoroughly...ie use alt config and test relay outputs can be controlled using these gpios

Reply:
now.I removed the EXTERN_GPIO function of PD3,PD4.